lib/services/workflow-api-service.js

```diff
@@ -155,7 +237,32 @@ function workflowApiServiceFactory(
     };
 
     WorkflowApiService.prototype.findActiveGraphForTarget = function(target) {
-        return taskGraphStore.findActiveGraphForTarget(target);
+        return waterline.graphobjects.find({ node: target })
+            .then(function(workflows){
+                return _.filter(workflows, function(wf) {
+                    return wf.active();
```

Comment:
@benbp Yeah, I agree a query makes sense here.  I think the thing that makes this a little confusing is that _status == pending is synonomous with "active".  Maybe we should add an active property to Constants.Task.States?

On the second point, I think the difference was the id property.  The raw Mongo document named this property "_id" whereas waterline omits the underscore.  There may be other differences too, but I'd need to look at it again.